blades/ws2811_blade.h

```diff
@@ -253,11 +253,16 @@ WS2811_Blade(WS2811PIN* pin,
       if (!current_style_ || !run_) {
 	loop_counter_.Reset();
 #ifdef BLADE_ID_SCAN_MILLIS
-	if (pin_->pin() == bladeIdentifyPin && ScanBladeIdNow()) {
-	  pin_->Enable(powered_);
-	  SLEEP(1);
-	}
-#endif      
+#if VERSION_MAJOR <= 5
+        if (pin_->pin() == bladeIdentifyPin && ScanBladeIdNow())
```

Comment:
for now I think we just replace bladeIdentifyPin with bladePin for everybody,  no need for an #ifdef

Reply:
ok this part is done.